File: tests/openmp_examples/tasking/taskloop_dep.1.f90
!$omp    taskloop nogroup
!$omp       task_iteration depend(inout: A(i)) depend(in: A(i-1))
!$omp    end taskloop
!$omp    taskloop grainsize(strict: 4) nogroup
!$omp       task_iteration depend(inout: A(i)) depend(in: A(i-4)) if(mod(i, 4) == 1 .or. i == n)
!$omp    end taskloop
!$omp    task depend(in: A(n))
!$omp    end task

File: tests/openmp_examples/tasking/taskloop_dep.1.c
#pragma omp taskloop nogroup
#pragma omp task_iteration depend(inout: A[i]) depend(in: A[i-1])
#pragma omp taskloop grainsize(strict: 4) nogroup
#pragma omp task_iteration depend(inout: A[i]) depend(in: A[i-4]) if ((i % 4) == 0 || i == n-1)
#pragma omp task depend(in: A[n-1])
